source/EvoPlanet/evoplanet.client/src/app/components/simulator/simulator.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="simulator-container">
+
+  
+  <a class="simulatortohome" aria-current="page" href="home">Go back to Homepage</a>
+  <canvas id="simulator-canvas" width="1850" height="820"></canvas>
```

Comment:
I don't think that width and height should be defined with fix pixel numbers. Any other window size, monitor size could break this setting, not to mention mobile devices.

Currently, I would suggest that width should be set to 100%. You can achieve the border with setting the margin to a few number of pixels.

The height is a bit trickier, and I don't have a solution in memory. It might be worth a little research on your part to set it more responsively than a fix number of pixels.